### PR TITLE
Increase sample count in voice message thumbnail

### PR DESCRIPTION
**Pull request by turt2live**
Fixes https://github.com/vector-im/element-web/issues/17817

Technically requires https://github.com/matrix-org/matrix-react-sdk/pull/6357 for sample sizing.

### CODE REVIEW

File: src/voice/Playback.ts [hunk 1]
```diff
@@ -92,6 +95,14 @@ export class Playback extends EventEmitter implements IDestroyable {
         return this.resampledWaveform;
     }
 
+    /**
+     * Stable waveform for representing a thumbnail of the media. Values are
+     * guaranteed to be between zero and one, inclusive.
+     */
+    public get waveformThumbnail(): number[] {
+        return this.thumbnailWaveform;
+    }
```
Comment (t3chguy) [inline src/voice/Playback.ts:104]: this is only ever read, we could make it `public readonly` and have TS guard it instead

**Review by t3chguy — APPROVED** [review]
LGTM otherwise

File: src/voice/Playback.ts (hunk 1 above)
Comment (turt2live) [reply src/voice/Playback.ts:104]: doh, of course. I'm honestly impressed at my naming here managing to avoid the conflict that would make it obvious.

Comment (t3chguy) [reply src/voice/Playback.ts:104]: At first, looking at the other file I was like wait that is totally backwards before I saw the getter lol

**Pull request merged**